package.json

```diff
@@ -391,7 +391,8 @@
           "description": "Specifies Lint tool name.",
           "enum": [
             "golint",
-            "gometalinter"
+            "gometalinter",
+            "megacheck"
```

Comment:
![image](https://user-images.githubusercontent.com/10286538/28242596-229b48bc-696d-11e7-9faa-760ded277ec1.png)

Finally got around to testing, (hello from Gophercon!), everything looks like it works ok for me without changes.